CHANGELOG.md

```diff
@@ -4,6 +4,49 @@ This document includes a curated changelog for each release. We also publish a c
 a [GitHub release](https://github.com/nginxinc/nginx-gateway-fabric/releases), which, by contrast, is auto-generated
 and includes links to all PRs that went into the release.
 
+## Release 1.4.0
+
+_August 20, 2024_
+
+FEATURES:
+
+- Support IPv6. [2190](https://github.com/nginxinc/nginx-gateway-fabric/pull/2190)
+- Add support for TLS Passthrough using TLSRoutes. [2356](https://github.com/nginxinc/nginx-gateway-fabric/pull/2356)
+- Support cross-namespace routing with TLSRoutes. [2379](https://github.com/nginxinc/nginx-gateway-fabric/pull/2379)
+- Added server_zone metrics for NGINX Plus users. [2360](https://github.com/nginxinc/nginx-gateway-fabric/pull/2360)
+
+BUG FIXES:
```

Comment:
Should we mention this:

Replace TODO route condition with an Accepted/False condition by @kate-osborn in https://github.com/nginxinc/nginx-gateway-fabric/pull/2228
?